src/inverseRobot.js

```diff
@@ -8,6 +8,22 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+  const robotEntries = Object.entries(robot);
+
+  for (let i = 0; i < robotEntries.length; i++) {
+    for (let j = i; j < robotEntries.length; j++) {
+      if (robotEntries[i][1] === robotEntries[j][1] && i !== j) {
+        return null;
+      }
+    }
+  }
```

Comment:
Don't write nested loops unless completely necessary.
Think on how to perform the same check in yout for ... of loop.